.github/workflows/dep-summaries.yml

```diff
@@ -90,19 +91,11 @@ jobs:
           output="${output//$'\n'/'%0A'}"
           output="${output//$'\r'/'%0D'}"
           echo "::set-output name=diff::${output}"
-
       - name: read dep-auditors list
         id: dep-auditors
-        if: ${{ steps.verify-lsr.outputs.diff || steps.verify-lec.outputs.diff }}
         run: |
           output=$(cat ./.github/dep-auditors.json | jq -r 'join(",")')
           echo "::set-output name=list::${output}"
-      - name: require dep-audit review if TCB deps changed
```

Comment:
You can't move this to the other job because only this job runs on push events. So this will remove the review requirement from the auto branch PR and so bors won't be able to enforce this.

The change above that removes the `if` line also can be restored if this remains here and you won't need to pass dep-auditor output around.

Reply:
It looks to me like both jobs fire on:

if: ${{ github.event_name == 'push' || github.actor != 'bors-libra' }} (well one is flipped from the other)
And the on: section applies to all jobs....

So they should both fire everywhere.

I was viewing calc summaries as a data provider, and annotate-pr as the worker.   I originally had 3 different jobs but that felt wasteful.    I could see mergeing both jobs in to one so bors wouldn't have to change.

Thoughts?

Reply:
Restoring the if line above would cause the original line 173 (if it were pointed at the right value) to fail, and the use of that output is already guarded by an if statement.

https://github.com/libra/libra/pull/6636/files#diff-877b1dac10c8b67e30963c98116d9dfa92a541bc15ea8d7ca78b3c8c80ad8700L171-L173

Reply:
And as you can see here if the failing step is moved back in to the calculate job, the annotation update job never gets fired off:  https://github.com/libra/libra/runs/1342165650?check_suite_focus=true

Reply:
Splitting the job in to three, calc, enforce, and update pr seem the best approach then.   But the enforcement step can't occur before another step/job that you want to run.

https://github.com/libra/libra/runs/1342269715

Reply:
but if calc fails, and enforce never runs, what will bors choose to do?